draft-ietf-quic-http.md

```diff
@@ -554,8 +554,11 @@ The following pseudo-header fields are defined for requests:
 
 All HTTP/3 requests MUST include exactly one value for the ":method", ":scheme",
 and ":path" pseudo-header fields, unless it is a CONNECT request ({{connect}}).
-An HTTP request that omits mandatory pseudo-header fields or contains invalid
-values for those fields is malformed ({{malformed}}).
+If the ":scheme" pseudo-header field identifies a scheme which has a mandatory
+authority component (including "http" and "https"), the request MUST contain
+either an ":authority" pseudo-header field or a "Host" header field; these
+fields MUST NOT be empty. An HTTP request that omits mandatory pseudo-header
```

Comment:
I find the wording `; these fields MUST NOT be empty.` to be slightly ambiguous because it just follows a sentence that says that they may be omitted. Perhaps something like `; additionally, if these headers are sent, their values MUST NOT be empty.`?

Reply:
Works for me.  While I was at it, we don't specify that they need to match, so I've added that, too.